.annotaterb.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
autogenerated file

Reply:
autogenerated but adjusted given the configs in `lib/tasks/auto_annotate_models.rake` right?

Reply:
correct!